internal/acceptance/volume_test.go

```diff
@@ -0,0 +1,143 @@
+package acceptance
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/acctest"
+)
+
+func TestUcAccVolumesResourceWithoutInitialOwnerAWSFullLifecycle(t *testing.T) {
+	GetEnvOrSkipTest(t, "TEST_BUCKET")
```

Comment:
Not sure if there is a better way for filtering specific cloud with UC and UC Volumes enabled.

Reply:
You could set a separate var on the env to filter on. Would this in its current form fail in any of our nightly test environments?

Reply:
> Would this in its current form fail in any of our nightly test environments?
No, this in its current form won't fail any of the nightly test environments because `"TEST_BUCKET"` only exists for one environment currently but this is fragile for short term (in case "TEST_BUCKET" is added to some other environment), in long term I think UC Volumes would be available in UC enabled prod workspaces so we might not required special filtering based on this.

Do you suggest creating a separate variable for this in short term or leave as is? We would need to remove that separate variable once Volumes is default UC enabled so it's an extra overhead.

Reply:
we don't `GetEnvOrSkipTest` in the beginning of the test.

Reply:
it's be skipped by `{env.TEST_BUCKET}`